_posts/2021-04-25-ocp-java11.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title:  'Experience Report: Taking the Online OCP Java SE 11 Developer Certification (1Z0-819)'
+author: karsten
+featuredImage: java11-omg
+excerpt: "OCP Java SE 11 Developer (1Z0-819) is the successor of previous OCA and OCP certifications
+combined in one. In this article you will find a personal experience report from taking the online
+exam, starting from preparation over study to the exam day.
+"
+categories: [Java, Certification]
+header:
+  text: OCP 1Z0-819 Experience Report
+  image: post
+---
+
+On April 23th 2021 I have done the OCP Java 11 Developer (1Z0-819) exam and passed. Oracle released this new certification
+in October 2020 as a successor of Java SE 11 Programmer I (1Z0-815) and Java SE 11 Programmer II (1Z0-816) combined in one.
+
+It was an intensive experience, and I wanted to share something about the motivation, preparation and the exam itself.
+The views expressed here are subjective. I try to give you a detailed insight why and how I started to prepare for the exam
+to the point where I performed the exam. Hopefully you will find some inspiration and tips for your own journey
+to get the certification.
+
+## Why doing the Java Certification?
+
+I'm now working with Java since my study days 25 years ago and got first in touch with the language in its 1.0 version, mainly developing small applets 
+and for studying computer languages. From my first day as a consultant back in 2000 I am programming with Java almost every day. Languages, language workbenches 
+and developing tools for developers have become a main part of my work, so my Java skills are solid.
+
+I never felt that it is necessary to have a Java certification in order to accomplish to work on complex Java based systems. I believe even for potential customers 
```

Comment:
...in order to ~~accomplish to work~~ **successfully work** on complex Java-based systems.